briskedu/brisk-learning
# Strengthen login experience

Summary:
This change makes the extension better handle things like login not being available on page load or when making a request. When the page loads, the revision check will return 401 if the user doesn't have a valid google auth already saved. If we receive 401, we show the user the login flow.  This also happens if they are logged out before page load and checking authorization.

https://app.asana.com/0/1203963408726580/1204099843417536

Test Plan:
1. Load page for new user and confirm 401 is returned
2. Confirm login menu shows up
3. Confirm login works and revision check is completed
4. Confirm if you reload, revision check succeeds on first try
5. Confirm if you revoke permissions while the page is loaded and click Check for AI Writing, you get a 401 and the login flow again
